keela-exe/mainwindow.cpp

```diff
@@ -67,11 +66,26 @@ MainWindow::MainWindow(): Gtk::Window() {
     container.add(num_camera_spin);
 
     show_trace_check.set_label("Show Traces");
-    trace_fps_spin.m_spin.set_adjustment(Gtk::Adjustment::create(125, 1, 1000, 1));
-    trace_fps_spin.set_sensitive(false);
+    trace_fps_spin.m_spin.set_adjustment(Gtk::Adjustment::create(DEFAULT_TRACE_FPS, 1, 1000, 1));
     trace_fps_spin.m_spin.signal_value_changed().connect(sigc::mem_fun(*this, &MainWindow::on_trace_fps_changed));
+    
+    trace_buffer_seconds_spin.m_spin.set_adjustment(Gtk::Adjustment::create(10, 1, 100, 1));
+    trace_buffer_seconds_spin.m_spin.signal_value_changed().connect(sigc::mem_fun(this, &MainWindow::on_trace_buffer_seconds_changed));
+
+    trace_clear_buffer_button.signal_clicked().connect(sigc::mem_fun(this, &MainWindow::on_trace_clear_buffer_button_clicked));
+
+    // These components are only relevant when traces are enabled
+    set_trace_dependent_sensitivities(false);
```

Comment:
Consider instead making trace_control_row_box a class member instead of a managed widget. Then you should be able to set the entire box to sensitive/not sensitive with a single function call.